apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,9 +618,9 @@ var ConversationView = {
         this.draft = Threads.get(threadId).getDraft();
 
         if (this.draft) {
-          this.draft.isEdited = false;
-
           Compose.fromDraft(this.draft);
+
+          this.draft.isEdited = false;
```

Comment:
actually, I think the "draft" mechanism should just be the model for Compose: currently, when sending, we need to extract the content from the DOM; but we should just send the current draft (granted it would itself come from the DOM because we use a contentEditable field).